spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/token/authentication/XsuaaAudienceValidatorTest.java

```diff
@@ -67,8 +68,7 @@ public void testSameClientIdWithoutAudience() {
 	public void testExtractAudiencesFromTokenScopes() {
 		Jwt token = new JwtGenerator()
 				.addScopes("test1!t1.read", "test2!t1.read", "test2!t1.write", ".scopeWithoutAppId").getToken();
-		List<String> audiences = new XsuaaAudienceValidator(serviceConfigurationSameClientId)
-				.getAllowedAudiences(token);
+		Set<String> audiences = XsuaaAudienceValidator.getAllowedAudiences(token);
```

Comment:
This is not required. Please keep existing tests as it is.

Reply:
Well, I don't agree to that end:
1. I must adjust the return type of the method in any case.
2. The method under test is declared as `static` - and hence should be accessed not through an instance, but better through the class.

I would be willing to revert item 2 (though I still think it's wrong), but reverting item 1 would break your test.

Please comment on what you want to have :)

Reply:
Hi @eaglerainbow,

You‘re right, thanks for the hint! I will rename the test afterwards...

Best regards,
Nena